components/docs-chef-io/content/habitat/get_started.md

```diff
@@ -0,0 +1,257 @@
++++
+title = "Get Started with Chef Habitat"
+gh_repo = "habitat"
+
+[menu]
+  [menu.habitat]
+    title = "Get Started"
+    identifier = "habitat/get_started"
+    parent = "habitat"
+    weight = 5
++++
+
+This getting started guide will show you how to use Chef Habitat to build and deploy a Node.js application.
+
+## Prerequisites
+
+Before getting started with this tutorial, you will need:
+
+- A [GitHub account](https://github.com/join)
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed locally (optional)
+- The [Chef Habitat CLI]({{< relref "/habitat/install_habitat" >}}) installed locally
+- An account on [Chef Habitat Builder]({{< relref "builder_account" >}})
+- A [profile on you Builder account]({{< relref "builder_profile" >}})
+
+### Create an origin and set up the Habitat CLI
+
+1. If you don't already have one, [create an origin]({{< relref "habitat/builder_origins" >}}) in your Chef Habitat Builder account. This origin will store the Habitat package that you create in this tutorial.
+
+1. If you haven't already done so, run `hab cli setup` and follow the instructions. This will configure the Hab CLI to work with your origin on the Habitat Builder SaaS.
+
+   You will need to provide the origin you created in your Builder account and a personal access token, which you can generate in your [Profile page in Habitat Builder](https://bldr.habitat.sh/#/profile).
```

Comment:
This actually will not be necessary based on the flow of this tutorial. You would only need it if you were uploading a hart file or promoting a build via the cli.

Reply:
Is this part required by the last part of this tutorial? I feel like I had to do this for 

https://deploy-preview-8838--chef-habitat.netlify.app/habitat/get_started/#manage-the-service-with-habitat-builder
or 
https://deploy-preview-8838--chef-habitat.netlify.app/habitat/get_started/#rebuild-the-service-from-your-github-repository